tests/helpers/linux/test_mm.py

```diff
@@ -134,3 +145,14 @@ def test_environ(self):
             proc_environ = f.read().split(b"\0")[:-1]
         task = find_task(self.prog, os.getpid())
         self.assertEqual(environ(task), proc_environ)
+
+    def test_slab_for_each(self):
+        proc_slabinfo_names = get_proc_slabinfo_names()
+        foreach_slab_names = [s.name.string_() for s in for_each_slab_cache(self.prog)]
+        self.assertEqual(proc_slabinfo_names, foreach_slab_names)
+
+    def test_find_slab_cache(self):
+        proc_slabinfo_names = get_proc_slabinfo_names()
+        for slab_name in proc_slabinfo_names:
```

Comment:
No need for this variable.

```suggestion
        for slab_name in get_proc_slabinfo_names():
```